### DISCUSSION TITLE
Art of constellation Puppis

### DISCUSSION THREAD

**Discussion by AndreLSilva1968** [Stellarium/stellarium#2318] [Q&A] [answered]
Hello everybody! In preparing a course about southern constellations using the Stellarium I repared in the art of the constellarion Puppis. It seems incorrect to me. The art seems to depict the prow or the "head" of the Argo Navis (the ship Argo). Puppis is the latin word for stern or "tail" of the ship. So, the entire figure seems drawn in a wrong direction. What you think?
![Argo-Navis](https://user-images.githubusercontent.com/100789231/156405022-8253b839-cd3e-43a3-aa7e-c0291b27ce98.png)

**Comment by gzotti** [comment, marked as answer]
Before being broken up into Carina, Vela and Puppis the constellation was just called "Argo Navis", and our artist apparently wanted to provide a more dramatic rendition of a ship, at cost of a directional reversal. Puppis should indeed be the back end of Argo. The classical atlases usually only show the back of a ship, sometimes with the front in clouds. Unfortunately we have no other image in the same artistic style. If you want to show the historical version, you will indeed need to show an image from some old atlas.

I cannot say who first described Argo as being half-covered in clouds. @sushoff?

**Reply by AndreLSilva1968** [reply]
Thank you for your answer! It is aways possible to show a figure from a old atlas, such as Hevelius Atlas Coelestium, and explain to audience that ancient observers saw the ship Argo in a reverse direction of that depictured in the art of Stellarium and that is the reason for the name "Puppis" (stern of the ship Argos).

**Comment by freakytalez8-afk** [comment]
[backwards solar system map of earth.html](https://github.com/user-attachments/files/24438245/backwards.solar.system.map.of.earth.html)
